pkg/op/auth_request_test.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/gorilla/schema"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-
 	httphelper "github.com/zitadel/oidc/v2/pkg/http"
 	"github.com/zitadel/oidc/v2/pkg/oidc"
 	"github.com/zitadel/oidc/v2/pkg/op"
```

Comment:
is the branch for service accounts mentioned on line 21 merged? if so can we remove the comments and execute the tests?

Reply:
I will investigate and create a new PR for that. The git blame is 2 years old from Livio ;)

Reply:
perfect, i approve this one

Reply:
https://github.com/zitadel/oidc/issues/335